packages/docusaurus-theme-common/src/index.ts

```diff
@@ -5,16 +5,29 @@
  * LICENSE file in the root directory of this source tree.
  */
 
+export {useThemeConfig} from './utils/useThemeConfig';
+
 export {
-  useThemeConfig,
   ThemeConfig,
-  Navbar,
-  NavbarItem,
-  NavbarLogo,
-  Footer,
-  FooterLinks,
-  FooterLinkItem,
-} from './utils/useThemeConfig';
+  ThemeNavbar,
+  ThemeNavbarItem,
+  ThemeNavbarLogo,
+  ThemeFooter,
+  ThemeFooterLinks,
+  ThemeFooterLinkItem,
+  /** @deprecated use ThemeNavbar */
+  ThemeNavbar as Navbar,
+  /** @deprecated use ThemeNavbarItem */
+  ThemeNavbarItem as NavbarItem,
+  /** @deprecated use ThemeNavbarLogo */
+  ThemeNavbarLogo as NavbarLogo,
+  /** @deprecated use ThemeFooter */
+  ThemeFooter as Footer,
+  /** @deprecated use ThemeFooterLinks */
+  ThemeFooterLinks as FooterLinks,
+  /** @deprecated use ThemeFooterLinkItem */
+  ThemeFooterLinkItem as FooterLinkItem,
+} from '@docusaurus/types';
```

Comment:
i'm unsure if we can rename it rn, i added prefix Theme as those names are used for component